server/email_templates/jira_account_denied.txt

```diff
@@ -7,6 +7,4 @@ the {project} project has chosen to deny the request. We therefore will not crea
 The following reason was given:
 {reason}
 
-If you wish to appeal this decision, you may do so either directly with the
-{project} project, or by contacting the ASF Infrastructure team at:
-users@infra.apache.org
+If you wish to appeal this decision, contact the {project} project directly.
```

Comment:
Would it work for all projects to add a `mailto` link to `private@{project}.apache.org` ?

Reply:
But then the next problem is: How to update the provided information ?
Me, as a PMC member, I cannot update any of the fields data.

Reply:
I don't think all projects want these jira requests handled via their private list. If a request has been denied, the person can do a little bit of legwork to find out how to properly contact the project.